examples/camera/Makefile

```diff
@@ -0,0 +1,269 @@
+# SPDX-License-Identifier: CC0-1.0
```

Comment:
I've changed a lot the makefiles included in the examples. Can you copy one of the updated ones?